scripts/circle_deploy.sh

```diff
@@ -10,26 +10,18 @@ git config --global user.name "tadeploy"
 git config --global user.email "devteam@technologyadvice.com"
 
 #
-# build
-#
-gulp docs
-
-#
 # generate changelog
 #
 echo "...generating changelog"
-github_changelog_generator $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME
+curl -X POST github-changelog-api.herokuapp.com/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME \
+ | json contents > CHANGELOG.md
```

Comment:
We post to the changelog api, grab the `contents` key from the json response and write it to the changelog.  Tested this locally and it works great.